### PR TITLE
[Laravel] Adding the "Getting started" guide for Laravel

### PR DESCRIPTION
**Pull request by gmoigneu**
## Why

Adding the Getting started guide for the Laravel framework on Upsun.

## What's changed

- Added `sites/friday/src/get-started/stacks/laravel/`
- Added the `env` configFile in `sites/friday/config/_default/params.yaml` that is replaced by `.environment`.

### CODE REVIEW

File: sites/friday/src/get-started/stacks/laravel/_index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Deploying Laravel on Upsun
+title: Deploying Laravel on {{% vendor/name %}}
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/_index.md:2]:
```suggestion
title: Deploying Laravel on Upsun
```

For some reason, the tags don't work here

File: sites/friday/src/get-started/stacks/laravel/blackfire.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Blackfire PHP SDK provides the following [integrations with
+Laravel](https://docs.blackfire.io/php/integrations/laravel/index):
+
+- [Laravel Artisan](https://docs.blackfire.io/php/integrations/laravel/artisan)
+- [Laravel Horizon and queue services](https://docs.blackfire.io/php/integrations/laravel/horizon)
+- [Laravel Tests](https://docs.blackfire.io/php/integrations/laravel/tests)
+- [Laravel Octane](https://docs.blackfire.io/php/integrations/laravel/octane)
+
+Please refer to the [Blackfire documentation](https://docs.blackfire.io/testing-cookbooks/tests#the-code-blackfire-yaml-code-file) to set up a `.blackfire.yml` configuration to enable custom [performance tests](https://blackfire.io/docs/testing-cookbooks/index)
+and automated [builds](https://blackfire.io/docs/builds-cookbooks/index).
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/blackfire.md:21]:
```suggestion
Please refer to the [Blackfire documentation](https://docs.blackfire.io/testing-cookbooks/tests#the-code-blackfire-yaml-code-file) to set up a `.blackfire.yml` configuration to enable custom [performance tests](https://docs.blackfire.io/testing-cookbooks/index)
and automated [builds](https://docs.blackfire.io/builds-cookbooks/index).
```

Typo on the Blackfire docs URL

File: sites/friday/src/get-started/stacks/laravel/crons.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Cron Jobs"
+weight: -105
+description: |
+    Understand how to configure Symfony cron jobs.
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/crons.md:5]:
```suggestion
    Understand how to configure Laravel cron jobs.
```

File: sites/friday/src/get-started/stacks/laravel/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+As Laravel relies heavily on environment variables to configure application services (such as the database or the mailer),
+the default configuration generated by `{{% vendor/cli %}} ify` automatically transport the [environment variables for all the services](#service-environment-variables) connected to your app via the `.environment` file.
+
+{{< note title="Tip" >}}
+
+You can tweak the `.environment` configuration to fit your needs and add specific project-level variables. If a variable needs to change per environment, use the ``{{% vendor/cli %}} variable` command to add and update them per environment.
+
+{{< /note >}}
+
+## Defaulkt environment variables
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/environment-variables.md:19]:
```suggestion
## Default environment variables
```

File: sites/friday/src/get-started/stacks/laravel/faq.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+---
+title: "FAQ"
+weight: 200
+description: |
+    Troubleshoot issue you might encounter using [Symfony](https://www.symfony.com/), a PHP framework on {{% vendor/name %}}.
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/faq.md:5]:
```suggestion
    Troubleshoot issue you might encounter using [Laravel](https://laravel.com/), a PHP framework on {{% vendor/name %}}.
```

File: sites/friday/src/get-started/stacks/laravel/faq.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+{{% vendor/cli %}} logs all
+  [app] [14-Aug-2020 10:52:27 UTC] [critical] Uncaught PHP Exception Exception: [...]
+  [app]
+  [php.access] 2020-08-14T10:52:27Z GET 500 2.386 ms 2048 kB 419.11% /
+  [access] 78.247.136.119 - - [14/Aug/2020:10:52:27 +0000] "GET / HTTP/1.1" 500 843 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.125 Safari/537.36"
+```
+
+If the error occurs on a preview environment,
+or on the main environment of a non-production project,
+you can also enable Symfony's dev/debug mode to inspect the cause of the error:
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/faq.md:75]:
```suggestion
you can also enable Laravel's dev/debug mode to inspect the cause of the error
via the `APP_DEBUG` [environment variable](./environment-variables.md) in your
`.environment` file or via [{{% vendor/cli %}} console](/development/variables.md):
```

File: sites/friday/src/get-started/stacks/laravel/faq.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+```bash
+# Enable debug mode
+{{% vendor/cli %}} env:debug
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/faq.md:79]:
```suggestion
export APP_DEBUG=1
```

File: sites/friday/src/get-started/stacks/laravel/faq.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+# Disable debug mode
+{{% vendor/cli %}} env:debug --off
```
Comment (thomasdiluccio) [inline sites/friday/src/get-started/stacks/laravel/faq.md:81]:
```suggestion
export APP_DEBUG=0
```

**Comment by chadwcarlson** [comment]
Content reviewed, updated, and merged in https://github.com/platformsh/platformsh-docs/pull/3899

**Pull request closed without merging**